### PR TITLE
DE17919: Added name field to Available Images

### PR DESCRIPTION
**Pull request by whywake-agarwal**
Summary: Added image name field to Available Images

Rally story/defect ID: DE17919

Pull request summary:
Added the image name as a field in the response of available images.

### CODE REVIEW

File: v1/api/swagger/components/schemas/AvailableImage.yaml [hunk 1]
```diff
@@ -23,6 +24,10 @@ properties:
     description: >-
       Second-level grouping of imaging services.  Typically references a
       specific OS or application.
+  Name:
+    type: string
+    example: HPE VMaaS VMware ESXi
```
Comment (mchuang3) [inline v1/api/swagger/components/schemas/AvailableImage.yaml:29]: This example conflicts with the Flavor, Version, and Description example, which is for an Ubuntu image, but the Name is an ESXi image. This makes the generated document look incorrect. Either change the Name example to "HPE Ubuntu 18.04", or change the examples to be an "ESXi" image.

**Review by dbozzato81 — COMMENTED** [review]
LGTM once Mike's comment is implemented.

File: v1/api/swagger/components/schemas/AvailableImage.yaml (hunk 1 above)
Comment (whywake-agarwal) [reply v1/api/swagger/components/schemas/AvailableImage.yaml]: [3a2c981](https://github.com/HewlettPackard/hpegl-metal-client/pull/90/commits/3a2c98106876073bb14f701c61a09cb3215b4ecc)

**Review by mchuang3 — APPROVED** [review]

**Comment by whywake-agarwal** [comment]
Closing this PR due to Action triggering issue. 
Updated PR: https://github.com/HewlettPackard/hpegl-metal-client/pull/95

**Pull request closed without merging**